rllib/models/tf/attention_net.py

```diff
@@ -549,10 +549,6 @@ def forward(
         model_out = self._logits_branch(self._features)
         return model_out, memory_outs
 
-    @override(ModelV2)
```

Comment:
I now get why you are deleting it, because the super() class has the same implementation. But I still think this should be overriden to explicitly return np.zeros(). The model creation stack does some weird magic with the Wrappers to "make it work" which is not transparent at all. Explicitly overriding it here would solve that issue imo.

Reply:
You are doing what I suggested a few lines below. Good! :)

Reply:
Sure thing!